Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -478,6 +478,7 @@ def condition(self, fragment, stepList, key, hasHarvest):
             ('2021' in key and fragment == "TTbar_14TeV"),
             ('2018' in key and fragment == "ZMM_13"),
             ('2021' in key and fragment == "ZMM_14"),
+            ('2026D88' in key and fragment == "TTbar_14TeV" and "PixelOnly" in self.suffix)
```

Comment:
Minor thing: One may need to update the comment in
https://github.com/cms-sw/cmssw/blob/master/Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py#L441-L445
to reflect the change introduced here.

Reply:
Thanks Phat, in the unlikely chance this PR does not get any further comment I'd leave it to the pixel track PR (: